README.md

```diff
@@ -51,6 +51,20 @@ fetch(options, function (err, content) {
 });
 ```
 
+#### Using one build with many verbs
+
+```javascript
+const behavior = {};
+const request = require("exp-fetch")(behavior);
+
```

Comment:
Worth mentioning that `behavior.httpMethod` will be overruled when using these to prevent any confusion?

Reply:
fixing